code-name-taran/src/com/spaghettisoft/component/belote/Bonuses.java

```diff
@@ -0,0 +1,49 @@
+package com.spaghettisoft.component.belote;
+
+public class Bonuses implements BelotConstants {

```

Comment:
This is very strange usage of an interface. You don't need too implement an interface to use the constants. It is generally unnatural to create an interface to hold constants only.